### PR TITLE
BRO: Urza, Lord Protector + The Mightstone and Weakstone

### PR DESCRIPTION
**Pull request by Suthro**
BRO: Add working scripts for two BRO cards following the recent Meld engine changes, + one token script.

- Urza, Lord Protector
- The Mightstone and Weakstone
- 1/1 colorless artifact Soldier creature token

Big thanks to Friarsol and TRT for the engine help!

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/urza_lord_protector_urza_planeswalker.txt [hunk 1]
```diff
@@ -0,0 +1,30 @@
+Name:Urza, Lord Protector
+ManaCost:1 W U
+Types:Legendary Creature Human Artificer
+PT:2/4
+S:Mode$ ReduceCost | ValidCard$ Artifact,Instant,Sorcery | Type$ Spell | Activator$ You | Amount$ 1 | Description$ Artifact, instant and sorcery spells you cast cost {1} less to cast.
+A:AB$ Meld | Cost$ 7 | ConditionPresent$ Card.Self+YouCtrl+YouOwn | ConditionCheckSVar$ X | Name$ Urza, Planeswalker | Primary$ Urza, Lord Protector | Secondary$ The Mightstone and Weakstone | SecondaryType$ Artifact | SpellDescription$ If you both own and control CARDNAME and an artifact named The Mightstone and Weakstone, exile them, then meld them into Urza, Planeswalker. Activate only as a sorcery.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/urza_lord_protector_urza_planeswalker.txt:6]: Missing SorcerySpeed

Comment (Suthro) [reply forge-gui/res/cardsfolder/upcoming/urza_lord_protector_urza_planeswalker.txt]: Fixed, thank you!

File: forge-gui/res/cardsfolder/upcoming/urza_lord_protector_urza_planeswalker.txt [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+Oracle:Artifact, instant and sorcery spells you cast cost {1} less to cast.\n{7}: If you both own and control Urza, Lord Protector and an artifact named The Mightstone and Weakstone, exile them, then meld them into Urza, Planeswalker. Activate only as a sorcery.
+
+ALTERNATE
+
+Name:Urza, Planeswalker
+ManaCost:no cost
+Colors:white,blue
+Types:Legendary Planeswalker Urza
+Loyalty:7
+S:Mode$ Continuous | EffectZone$ Battlefield | Affected$ Card.Self | AddHiddenKeyword$ CARDNAME's loyalty abilities can be activated twice each turn rather than only once | CharacteristicDefining$ True | Description$ You may activate the loyalty abilities of CARDNAME twice each turn rather than only once.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/urza_lord_protector_urza_planeswalker.txt:20]: Not CharacteristicDefining

Comment (Suthro) [reply forge-gui/res/cardsfolder/upcoming/urza_lord_protector_urza_planeswalker.txt]: Forgot to remove during testing, thanks!

**Review by tool4ever — APPROVED** [review]

**Pull request merged**